src/client.ts

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable */
```

Comment:
does this need to be checked into git?

I guess this is generated by `make test`?